app/src/main/java/com/example/atheneum/fragments/OwnerPageFragment.java

```diff
@@ -31,10 +56,40 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
             mainActivity = (MainActivity) getActivity();
             // set action bar title
             mainActivity.setActionBarTitle(context.getResources().getString(R.string.owner_page_title));
-
         }
 
+        retrieveBooks();
+
         return this.view;
     }
 
+    /**
+     * Retrieve books from Firebase.
+     *
+     * See: https://stackoverflow.com/questions/37902635/no-setter-field-for-warning-firebase-database-retrieve-data-populate-listview
+     */
+    public void retrieveBooks() {
+        FirebaseUser firebaseUser = FirebaseAuth.getInstance().getCurrentUser();
+        final FirebaseDatabase db = FirebaseDatabase.getInstance();
+        String dbOwnerBooksPath = getString(R.string.db_ownerCollection) + "/" + firebaseUser.getUid().toString();
+        DatabaseReference ref = db.getReference(dbOwnerBooksPath);
```

Comment:
Try getting the reference by using: `
```
DatabaseRef ref = db.getReference.child(getString(R.string.db_ownerCollection)).child(firebaseUser.getUid());
```

I've never had to convert firebaseUser.getUid() to a string to query firebase so maybe that was what the problem was.